source/collection-management/tests/timeseries-collection.yml

```diff
@@ -0,0 +1,57 @@
+description: "timeseries-collection"
+
+schemaVersion: "1.0"
+
+runOnRequirements:
+  - minServerVersion: "5.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name ts-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name test
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents: []
+
+tests:
+  - description: "createCollection with all options"
+    operations:
+      - name: dropCollection
+        object: *database0
+        arguments:
+          collection: *collection0Name
+      - name: createCollection
+        object: *database0
+        arguments:
+          collection: *collection0Name
+          timeseries: &timeseries0
+            timeField: "time"
+            metaField: "meta"
+            expireAfterSeconds: 604800
```

Comment:
Optional: When playing around with this, it surprised me is that `_id` no longer has a uniqueness constraint in a timeseries collection.

```
> db.runCommand({create: "timecoll", timeseries: {timeField: "t"}})
{
	"ok" : 1,
	"$clusterTime" : {
		"clusterTime" : Timestamp(1621606645, 2),
		"signature" : {
			"hash" : BinData(0,"AAAAAAAAAAAAAAAAAAAAAAAAAAA="),
			"keyId" : NumberLong(0)
		}
	},
	"operationTime" : Timestamp(1621606645, 2)
}
> db.timecoll.insert({_id:1, t: new Date()})
WriteResult({ "nInserted" : 1 })
> db.timecoll.insert({_id:1, t: new Date()})
WriteResult({ "nInserted" : 1 })
> db.timecoll.find().pretty()
{ "t" : ISODate("2021-05-21T14:17:31.341Z"), "_id" : 1 }
{ "t" : ISODate("2021-05-21T14:17:31.728Z"), "_id" : 1 }
```

I do not think this impacts driver API or CRUD results, so it does not strictly warrant additional tests. But since this is the first time that assumption no longer holds, if it is straightforward, it might be worthwhile to have a copy of this test with an `InsertMany` and `Find` using two documents with a duplicate `_id`.

Reply:
Ah good find, it does look like they support duplicate `_id` fields. I've added another test for `InsertMany` with duplicate `_id`s.